helpers_test.go

```diff
@@ -75,3 +76,36 @@ func Test_inspectHelper(t *testing.T) {
 	o := inspectHelper(s)
 	r.Contains(o, "Ringo")
 }
+
+func Test_envHelper(t *testing.T) {
+	envy.Temp(func() {
+		r := require.New(t)
+		envy.Set("TEST_KEY", "TEST_VALUE")
+
+		value, err := envHelper("TEST_KEY")
+		r.Nil(err)
+		r.Equal("TEST_VALUE", value)
+	})
+}
+
+func Test_envHelperMissing(t *testing.T) {
+	r := require.New(t)
+	_, err := envHelper("MISSING_TEST_KEY")
+	r.NotNil(err)
+}
+
+func Test_envOrHelper(t *testing.T) {
```

Comment:
You should probably change these tests to be a bit more like integration tests. See https://github.com/gobuffalo/plush/blob/master/plush_test.go for examples on how to do that.